www/_posts/2023-05-22-cordova-cli-12.0.0.md

```diff
@@ -0,0 +1,103 @@
+---
+layout: post
+author:
+    name: Bryan Ellis
+title:  "Cordova CLI 12.0.0 & Internal Packages Released!"
+categories: news
+tags: news releases
+---
+
+We are happy to announce that we have just released a major update to our Cordova CLI!
+
+* [cordova@12.0.0](https://www.npmjs.com/package/cordova)
+
+In this CLI release, it also includes the latest internal libraries:
+
+* [cordova-lib@12.0.1](https://www.npmjs.com/package/cordova-lib)
+* [cordova-create@5.0.0](https://www.npmjs.com/package/cordova-create)
+* [cordova-common@5.0.0](https://www.npmjs.com/package/cordova-common)
+* [cordova-fetch@4.0.0](https://www.npmjs.com/package/cordova-fetch)
+* [cordova-serve@4.0.1](https://www.npmjs.com/package/cordova-serve)
+
+
+## Release Highlights
+
+This release drops Nodejs 14 support. The minimum supported version that Cordova requires is greater than or equal to 16.13.0.
+
+In all releases, we have updated all npm packages to the possible latest release that Cordova can support.
+
+We have removed the deprecated platform **OSX** & **Windows** from the platform listing and removed all platform pinnings so that future platform releases are available to use immediately.
```

Comment:
> ... and removed all platform pinnings so that future platform releases are available to use immediately.

I feel like this should be highlighted as a breaking behaviour change. If people relied on the old behaviour to install a particular major version as pinned by Cordova, then the new behaviour would break that assumption. So it should be clear that a change in their workflow might be needed here if they want to maintain an old behaviour, e.g. they should pin to a particular version themselves if that is the desired action.